spec/features/parental_consent_inexact_auto_match_spec.rb

```diff
@@ -91,6 +94,7 @@ def when_a_parent_gives_consent_with_three_of_four_fields_matching_the_cohort
 
   def and_the_nurse_checks_the_consent_responses
     perform_enqueued_jobs
+    perform_enqueued_jobs
```

Comment:
```suggestion
    2.times { perform_enqueued_jobs }
```

Might be worth a comment just explaining why it has to be called twice.